### PR TITLE
Feat/fighterinput

### PR DESCRIPTION
**Pull request by Ricefab02**

### CODE REVIEW

File: src/components/FighterInput.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+
+class MyFighterInput extends Component {
```
Comment (rocambille) [inline src/components/FighterInput.jsx]: I agree it's yours, but you can drop the "My" ;)

File: src/components/FighterInput.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  handleChange(event) {
+    this.setState({ fighter: event.target.value });
+  }
+
+  render() {
+    return (
+      <div>
+         <label htmlFor="fighter">Fighter :</label>
+        <input
```
Comment (rocambille) [inline src/components/FighterInput.jsx]: fix indentation. did you install eslint extension in your vs code?

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/components/FighterInput.jsx (hunk 1 above)
Comment (Ricefab02) [reply src/components/FighterInput.jsx]: ok correction and commit done

File: src/components/FighterInput.jsx (hunk 2 above)
Comment (Ricefab02) [reply src/components/FighterInput.jsx]: ok correction and commit done

**Pull request merged**